cmd/influx/write.go

```diff
@@ -151,13 +150,17 @@ func (writeFlags *writeFlagsType) createLineReader(cmd *cobra.Command, args []st
 
 	// #18349 allow URL data sources, a simple alternative to `curl -f -s http://... | influx write ...`
 	if len(writeFlags.URLs) > 0 {
-		client := http.Client{Timeout: 5 * time.Minute}
+		client := http.DefaultClient
 		for _, addr := range writeFlags.URLs {
 			u, err := url.Parse(addr)
 			if err != nil {
 				return nil, csv2lp.MultiCloser(closers...), fmt.Errorf("failed to open %q: %v", addr, err)
 			}
-			resp, err := client.Get(addr)
+			req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, addr, nil)
```

Comment:
The default infinite timeout should be here from beginning, blame on me :) IMHO it would be probably better to require a context as a new parameter of `createLineReader` function and reuse the context with signals that is created in https://github.com/influxdata/influxdb/blob/9c779443d0c2947d6473f5599e1a7899c393cc00/cmd/influx/write.go#L295

Reply:
@sranka good points. I'm not super concerned about it as the OS will clean it all up, but is still something worth adding 👍